build/commands/lib/config.js

```diff
@@ -367,19 +362,12 @@ Config.prototype.enableCDMHostVerification = function () {
   } else {
     console.log('Widevine cdm host verification is disabled')
   }
-  return enable
-}
-
-Config.prototype.isAsan = function () {
-  if (this.is_asan) {
-    return true
-  }
-  return false
+  return enable ? true : undefined
```

Comment:
`enable_cdm_host_verification =
      enable_library_cdms && (is_mac || is_win) && is_chrome_branded &&
      !is_chrome_for_testing_branded` in media_options.gni which is always false for us because of `is_chrome_branded`